graphdatascience/gds_session/gds_sessions.py

```diff
@@ -45,7 +45,11 @@ def __init__(self, ds_connection: AuraAPICredentials) -> None:
         )
 
     def get_or_create(
-        self, session_name: str, size: SessionSizeByMemory, db_connection: DbmsConnectionInfo
+        self,
+        session_name: str,
+        size: SessionSizeByMemory,
+        db_connection: DbmsConnectionInfo,
+        region: Optional[str] = None,
```

Comment:
this is only needed while we have AuraDS based sessions. Afterwards we dont want to allow this actually